### PR TITLE
Add info about cloud storage per plugin

### PR DESCRIPTION
**Pull request by melcorr**
I am trying to set... expectations a bit? 

The motivation behind this is to try and express which plugins can be relied on not only to test but to test and keep cloud storage options working. 

@mdellweg has explained that it is tested in a wider range of plugins, but I want to list the plugins where any issues are reliably fixed. I suppose I am trying to express where the user might expect *some* degree of certainty. 

As part of #361

### CODE REVIEW

File: content-plugins.md [hunk 1]
```diff
@@ -34,6 +32,17 @@ At the moment, these plugins have the following coverage:
 * **Container**: Sync workflow
 * **Python**: Sync workflow
 
+## Cloud Storage
+
+It is possible to configure Pulp to use
+[cloud storage](https://docs.pulpproject.org/pulpcore/installation/storage.html).
+However, plugins can introduce changes that are incompatible with, for example, S3 requirements.
+The Pulp Ansible and Container plugins are regularly tested to ensure they remain compatible but the
```
Comment (mdellweg) [inline content-plugins.md]: Debian tests with S3, too.

Comment (mdellweg) [reply content-plugins.md]: So does RPM

Comment (mdellweg) [reply content-plugins.md]: and Python

File: content-plugins.md [hunk 2]
```diff
@@ -34,6 +32,17 @@ At the moment, these plugins have the following coverage:
 * **Container**: Sync workflow
 * **Python**: Sync workflow
 
+## Cloud Storage
+
+It is possible to configure Pulp to use
+[cloud storage](https://docs.pulpproject.org/pulpcore/installation/storage.html).
+However, plugins can introduce changes that are incompatible with, for example, S3 requirements.
+The Pulp Ansible, RPM, File, Debian, Python, and Container plugins are regularly tested to ensure they remain compatible but the
+level of coverage is lower for other plugins.
```
Comment (daviddavis) [inline content-plugins.md]: The formatting here is a bit off.

```suggestion
It is possible to configure Pulp to use [cloud
storage](https://docs.pulpproject.org/pulpcore/installation/storage.html). However, plugins can 
introduce changes that are incompatible with, for example, S3 requirements. The Pulp Ansible, RPM,
File, Debian, Python, and Container plugins are regularly tested to ensure they remain compatible
but the level of coverage is lower for other plugins.
```

File: content-plugins.md [hunk 3]
```diff
@@ -7,8 +7,8 @@ toc: true
 
 ## Content Plugins
 
-As part of the Pulp installation, you must add a content plugin for each content type that you want to
-manage.
+As part of the Pulp installation, you must add a content plugin for each content type that you want
+to manage.
 The following sections contain information about the available content plugins for both Pulp 2 and
 Pulp 3. If you do not find the plugin for the content type you want to manage, consider
 [writing a plugin](https://docs.pulpproject.org/plugins/plugin-writer/index.html).
```
Comment (daviddavis) [inline content-plugins.md]:
```suggestion
As part of the Pulp installation, you must add a content plugin for each content type that you want
to manage. The following sections contain information about the available content plugins for both
Pulp 2 and Pulp 3. If you do not find the plugin for the content type you want to manage, consider
[writing a plugin](https://docs.pulpproject.org/plugins/plugin-writer/index.html).
```

**Review by daviddavis — APPROVED** [review]

**Comment by melcorr** [comment]
@daviddavis where can I formally challenge the relevance of this 100 char rule ? :-)

**Comment by daviddavis** [comment]
We've discussed it before at open floor but I'd probably say post a thread on Github Discussions?

**Comment by melcorr** [comment]
> We've discussed it before at open floor but I'd probably say post a thread on Github Discussions?

Thanks!

**Pull request merged**